mathics/builtin/statistics/orderstats.py

```diff
@@ -145,6 +146,7 @@ def ranked(i):
 
 class Quartiles(Builtin):
     """
+    <url>:Quartile: https://en.wikipedia.org/wiki/Quartile</url> (<url>:WMA: https://reference.wolfram.com/language/ref/Quartiles.html</url>)
```

Comment:
Somes we have WMA and sometimes WMA link.

Reply:
I tried to follow the existing pattern, but sometimes the pattern was not there...

Reply:
> I tried to follow the existing pattern, but sometimes the pattern was not there...

Here's a pattern to follow. If WMA is not in parenthesis add "link", if it is in parenthesis as it is here, no "link" added. 

So this one is okay. But I think I have seen "WMA link" in parenthesis elsewhere and it is something to go over for another iteration.